llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1236,20 +1235,21 @@ class LoopVectorizationCostModel {
   /// the loop.
   void collectInstsToScalarize(ElementCount VF);
 
-  /// Collect Uniform and Scalar values for the given \p VF.
+  /// Collect Uniforms, Scalars, and Insts to Scalarize for the given \p VF.
   /// The sets depend on CM decision for Load/Store instructions
   /// that may be vectorized as interleave, gather-scatter or scalarized.
   /// Also make a decision on what to do about call instructions in the loop
   /// at that VF -- scalarize, call a known vector routine, or call a
   /// vector intrinsic.
-  void collectUniformsAndScalars(ElementCount VF) {
+  void collectNonVectorized(ElementCount VF) {
```

Comment:
while you are changing the name, it would be good to update to clarify that this does ore than just collecting the values that won't need widening, it also sets various widening decisions.

Reply:
Can you suggest a good name? I can think of `collectNonVectorizedSetWideningDecisions`?